include/irr/asset/IImageFilter.h

```diff
@@ -16,20 +16,76 @@ namespace irr
 namespace asset
 {
 
-// runtime polymorphic
+//! Base class for general filters with runtime polymorphism.
+/*
+	Filters can execute various actions basing on input image
+	to get an output image.
+	Available filters are as following:
+
+	- Fill Filter
+	- Copy Filter
+	- Flatten Filter
+	- Convert Filter
+	- Swizzle && Convert Filter
+	- Utility Filter
```

Comment:
Utility filter isn't an actual filter you can use, its a base for some of the filters so its less repeating code

Reply:
deleted